Library/ViewModels/BackerDashboardViewModel.swift

```diff
@@ -118,9 +119,33 @@ public final class BackerDashboardViewModel: BackerDashboardViewModelType, Backe
         .materialize()
     }
 
+    let graphUserEvent = Signal.merge(
+      self.projectSavedProperty.signal.ignoreValues()
+        .ksr_debounce(.seconds(1), on: AppEnvironment.current.scheduler),
+      self.viewWillAppearProperty.signal.ignoreValues()
+    )
+    .switchMap { _ in
+      AppEnvironment.current
+        .apiService.fetchGraphUserSelf()
+        .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+        .materialize()
+    }
+
     let user = fetchedUserEvent.values()
 
+    let updatedUserWithSavedProjectsCount = graphUserEvent.values()
+      .map { $0.me }
+
     self.updateCurrentUserInEnvironment = user.skip(first: 1)
+      .takePairWhen(updatedUserWithSavedProjectsCount)
+      .map { user, graphEventUser in
+        var updatedUserWithStarredProjectsCount = user
+
+        updatedUserWithStarredProjectsCount.stats.starredProjectsCount = graphEventUser.stats
+          .starredProjectsCount
```

Comment:
it might be nice, at some point, to update the naming here since we call them saved projects and use a heart icon.

Reply:
Yea definitely. We have this big gap in our models for `User` and `GraphUser`. A lot of pages use the v1 endpoints that map to `User`. We'll at some point convert `User` to gql endpoints app-wide...so good opportunity to do it then.